components/_patterns/00-base/global/base/_pl-base.scss

```diff
@@ -78,5 +117,47 @@
       height: 100px;
       margin-bottom: 0.3em;
     }
+
+    + p {
+      @include wrapper;
+    }
+  }
+}
+
+// Elements that aren't typically affected by the wrapper,
+// but we want to line up in PL.
+// Mostly base and atom components.
+.sg-pattern-example {
```

Comment:
@evanmwillhite I'm not even done listing these here, and I'm already annoyed. I'm not sure this is a viable solution. Can you take a look at this and suggest a better way to do this?

I'm selecting each one, instead of just doing
```
.sg-pattern-example {
  @include wrapper;
}
```

because if we do that ^ and then also add the wrapper mixin to the actual elements that need it. e.g. `.card-grid`, there's a double wrapper thing going on that adds unwanted padding, and is extra weird on mobile.

Maybe we can talk about this tomorrow/friday.